src/server.c

```diff
@@ -3710,6 +3710,60 @@ int commandCheckArity(client *c, sds *err) {
     return 1;
 }
 
+int findDuplicatesOptions(client *c, const char *token) {
+    int matched = 0;
+    for (int j = 2, k = c->argc - 1; j <= k; j++, k--) {
+        if (!strcasecmp(token, c->argv[j]->ptr)) {
```

Comment:
How do we handle the case if another field has the same value as the `token` ?

Reply:
Thank you for the feedback. I realized that I overlooked a case that complicates separating or identifying the value and option (token) within the processCommand function. I'll explore different ways to address this issue. If you have any suggestions or ideas on how to handle this, please feel free to share them. Thanks again!